### ISSUE TITLE
Zoom value: Fit page

### ISSUE THREAD

**Issue by mt633** [opensheetmusicdisplay/opensheetmusicdisplay#419]
Would it be possible to make OSMD render the MusicXML file so that the zoom value is calculated to fit what is rendered as one page in e.g. MuseScore within the screen height?

**Comment by sschmidTU** [comment]
That could definitely be an option. Maybe it should even be the default option, though some people might prefer not to have the initial scale depend on the length of the score (in case it's shorter than a standard size page).
We wouldn't even have to render twice, right? Because we can tell Vexflow how large its canvas should be, no need to estimate from a previous render how much we need to scale.

**Comment by bneumann** [comment]
I wonder, with the autoresize option and the media tag set to responsive it gives me only the size my phone can handle.
So basically we already have that. I was just fiddling with it while creating the react component, but maybe the setup is different. Like @zwiebelspaetzle with the flexbox?

**Comment by mt633** [comment]
Not sure what you mean @bneumann. As far as I can see, auto-resize only works for the width of the screen, not for the height. I would like the option to fit one page (where there are page breaks in the XML) on the screen on load time, even if that means I manually need to zoom in from there to read. But maybe I misunderstood you?

**Comment by matt-uib** [comment]
@mt633 yes i guess there was a misunderstanding here. Currently we have no page layout like e.g. rendering the systems to Letter or A4 pages. We only take the width of the "page" into account and make an "endless" page regarding the height. But soon we want to add support for a real page layout which is also needed for printing. Then your request makes absolutely sense and we will try to add this option then.

**Comment by mt633** [comment]
Thanks for the clarification @matt-uib! It depends on #372 then I suppose.

**Comment by sschmidTU** [comment]
We have pagination and PDF export now, so i guess the only thing missing is an option to make OSMD fit a score on x pages (in your case x=1). In that case, we probably have to calculate a scale factor to get an exact page fit.
